.travis.yml

```diff
@@ -16,17 +16,18 @@ env:
   - ENCRYPTION_LABEL: "898e0934cf65"
   - COMMIT_AUTHOR_EMAIL: "janos.sallai@vanderbilt.edu"
 
+
+
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install md5sha1sum; fi
   - source devtools/travis-ci/install.sh
-  - conda config --add channels omnia
-  - conda config --add channels janschulz
-  - conda config --add channels bioconda
-  - conda config --add channels glotzer
-  - conda config --add channels mosdef
-
-script:
-  - conda build --python $PYTHON_VERSION devtools/conda-recipe
+  - conda config --set always_yes yes --set changeps1 no
```

Comment:
What is the purpose of the `changeps1` flag here? I've never seen this being necessary.